### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by JohnVeness** [maks/viper-edit#7]
Thanks for taking on this app. Are you planning to submit it for listing on F-Droid?

**Comment by maks** [comment]
Hi, I hadn't planned to, but you are welcome to do so if you would like to.

**Comment by fm-sys** [comment]
As the "old app" is still listed on F-Droid, it might make sense to retarget the fdroid entry to this repo... 

https://f-droid.org/packages/com.maskyn.fileeditorpro/

**Comment by maks** [comment]
@fm-sys I'd prefer to keep this port distinct from the original file editor pro, so I think a new entry would be better.

**Comment by maks** [comment]
I should clarify that I'm happy to have ViperEdit listed on F-droid, I just don't have the time to look into doing that at the moment, so I'm happy for someone else to go ahead and do that since afaik it doesn't have to be myself as the maintainer of ViperEdit that needs to be the one to do it.

**Comment by fm-sys** [comment]
> @fm-sys I'd prefer to keep this port distinct from the original file editor pro, so I think a new entry would be better.

How is it done for Google Play? Does ViperEdit has a different application ID than the old no longer maintained TurboEditor?

If so, I agree that ViperEdit should have its own entry. Even more, it even needs to be like that in this case...

**Comment by fm-sys** [comment]
> Does ViperEdit has a different application ID than the old no longer maintained TurboEditor?

Ok, seems to be the case:
com.maskyn.fileeditorpro
com.manichord.viperedit

BTW, while we are on it: Shouldn't we change the application name to "ViperEdit"?

![Screenshot_20210801_110501_com huawei android launcher](https://user-images.githubusercontent.com/64581222/127765536-8adb1251-479b-406d-812e-d930570fefb4.png)

**Comment by maks** [comment]
yes thats correct, as you've noticed the app id is different on Google Play. 
Oh and yes, thanks for pointing out the need to update the name.
